README.md

```diff
@@ -43,11 +43,13 @@ allowing you to choose what files to overwrite.
 These instructions will walk you through how to create a napari plugin. It uses an application called copier
 to prompt you for configuration input and does the work of creating a functioning, distributable plugin from your
 source code.
+Both options install the [Copier](https://copier.readthedocs.io/en/stable/) application,
+the [jinja2-time](https://pypi.org/project/jinja2-time/) extension,
+and the napari plugin engine [npe2](https://github.com/napari/npe2) to help validate your new plugin is configured correctly.
 
-### Step 1: Install the template tools.
+### [Option A]:  Use an environment to manage and utilize copier
```

Comment:
For simplicity, it may be better to swap the order of the two options: put uv approach first and virtual environment second.

I would change the title to [Option A (or B)]: Use a virtual or conda environment to run the plugin template